src/Containers/KitchenCard/KitchenCardItems.tsx

```diff
@@ -36,9 +31,18 @@ export const KitchenCardItems: React.FC<KitchenCardItemsProps> = ({
                         >
                             {item.name}
                         </Heading>
-                        {modifiers[index].map(
-                            (fil): React.ReactElement => (
-                                <Paragraph margin="0 0 0 20px">{fil}</Paragraph>
+                        {item.modifiers.map((modifier) =>
+                            modifier.choices.map(
+                                (choice: Choice): React.ReactElement => (
+                                    <Paragraph margin="5px 0 0 20px">
+                                        {`${
+                                            choice.choice_type ===
+                                            ModifierChoiceTypeEnum.DEFAULT
+                                                ? ''
+                                                : choice.choice_type
+                                        } ${choice.name}`}
+                                    </Paragraph>
+                                ),
```

Comment:
Don't have inline functions inside your react tree, keep your react tree extremely small < 10 - 15 lines and use functions to do complex logic